src/Bicep.Cli/Program.cs

```diff
@@ -144,6 +150,32 @@ private void BuildManyFilesToStdOut(IDiagnosticLogger logger, string[] bicepPath
                 writer.WriteEndArray();
             }
         }
+
+        private static string ReadFile(string path)
+        {
+            try
+            {
+                return File.ReadAllText(path);
+            }
+            catch (Exception exception)
+            {
+                // I/O classes typically throw a large variety of exceptions
+                // instead of handling each one separately let's just trust the message we get
+                throw new BicepException(exception.Message, exception);
+            }
+        }
+
+        private static FileStream CreateFileStream(string path)
+        {
+            try
+            {
+                return new FileStream(path, FileMode.Create, FileAccess.ReadWrite, FileShare.None);
+            }
+            catch (Exception exception)
+            {
+                throw new BicepException(exception.Message, exception);
```

Comment:
Same question here. Can we trust the exception message to contain the file path?

Reply:
Same answer.